app/javascript/dashboard/modules/widget-preview/components/WidgetBody.vue

```diff
@@ -1,24 +1,50 @@
 <template>
-  <div class="conversation--container">
-    <div class="conversation-wrap">
-      <div class="message-wrap">
-        <div class="user-message-wrap">
-          <div class="user-message">
-            <div class="message-wrap">
-              <div class="chat-bubble user">
-                <p>Hello</p>
+  <div>
+    <div v-if="config.isDefaultScreen" class="availabilty-container">
+      <div class="availabilty-info">
+        <div class="team-status">
+          {{
+            config.isOnline
+              ? $t('INBOX_MGMT.WIDGET_BUILDER.BODY.TEAM_AVAILABILITY.ONLINE')
+              : $t('INBOX_MGMT.WIDGET_BUILDER.BODY.TEAM_AVAILABILITY.OFFLINE')
+          }}
+        </div>
+        <div class="reply-wait-message">
+          {{ config.replyTime }}
+        </div>
+      </div>
+      <div class="user-thumbnail-box">
+        <div class="user-thumbnail">
+          <span>J</span>
+        </div>
+      </div>
```

Comment:
Can we use our existing thumbnail component here with `size 40px`, so we can remove the CSS used here?

Reply:
Done